src/collective/solr/timeout.py

```diff
@@ -1,39 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
@tomgross @do3cc why did we remove the timeout functionality here? The configlet test fails because we removed the functionality but kept the test.